yarn.lock

```diff
@@ -7967,7 +7967,7 @@ fast-glob@^2.2.6:
     merge2 "^1.2.3"
     micromatch "^3.1.10"
 
-fast-glob@^3.1.1:
+fast-glob@^3.1.1, fast-glob@^3.2.7:
```

Comment:
I already use glob in other scripts, is fast-glob any better?

Reply:
I can't see glob as project dev dependency. You could use it because other modules install it. I think it will be safer to add it to our package json, and then I can use it in this script also. What do you think?

Reply:
Oh defintely, let’s add glob explicitly then